### PR TITLE
Add auto

### PR DESCRIPTION
**Pull request by guern1kn**
## What does this PR do?
Remove banner, swap Spark logo, add icon & link guidelines, add Rocket Auto to footer examples.

### Associated Issue
Fixes:
2610668
2602841
2572890
2610669

### Documentation
 - [x] Update Spark Docs

### Code
 - [x] Build Component in HTML
 - [x] Build Component in Angular
 - [x] Build Component in React
 - [x] Unit Testing in HTML with `npm run test` in `html/` (100% coverage, 100% passing)
 - [x] Unit Testing in Angular with `npm run test` in `angular/` (100% coverage, 100% passing)
 - [x] Unit Testing in React with `npm run test` in `react/` (100% coverage, 100% passing)

### Browser Testing (current version and 1 prior)
  - [x] Google Chrome
  - [ ] Google Chrome (Mobile)
  - [ ] Mozilla Firefox
  - [ ] Mozilla Firefox (Mobile)
  - [ ] Microsoft Internet Explorer 11 (only this specific version of IE)
  - [ ] Microsoft Edge
  - [ ] Apple Safari
  - [ ] Apple Safari (Mobile)

### CODE REVIEW

File: angular/projects/spark-angular/src/lib/components/sprk-footer/sprk-footer.stories.ts [hunk 1]
```diff
@@ -146,6 +146,14 @@ export const defaultStory = () => ({
             'https://spark-assets.netlify.app/rocket-homes-white.svg',
           imgAlt: 'Rocket Homes Logo',
           analyticsString: 'link-3'
+        },
+        {
+          text: 'Find and buy the perfect car or truck from thousands of vehicles, all in one marketplace',
```
Comment (afebbraro) [inline angular/projects/spark-angular/src/lib/components/sprk-footer/sprk-footer.stories.ts:151]: Does this need a period at the end?

File: react/src/components/footer/SprkFooter.stories.js [hunk 2]
```diff
@@ -60,6 +60,15 @@ export const defaultStory = () => (
           element: 'a',
           mediaHref: '#nogo',
         },
+        {
+          mediaType: 'image',
+          src: 'https://spark-assets.netlify.app/rocket_auto.svg',
+          altText: 'Rocket Auto Logo',
+          description:
+            'Find and buy the perfect car or truck from thousands of vehicles, all in one marketplace',
```
Comment (afebbraro) [inline react/src/components/footer/SprkFooter.stories.js:68]: Does this need a period at the end?

**Review by afebbraro — APPROVED** [review]
Thank you! looks great, i just added a comment about if there was a missing period at the end of a sentence.

File: html/components/footer.stories.js [hunk 3]
```diff
@@ -111,6 +111,24 @@ export const defaultStory = () => {
...
+                </a>
+              </div>
+
+              <p
+                class="
+                  sprk-o-Stack__item
+                  sprk-b-TypeBodyFour
+                  sprk-c-Footer__text
+              ">
+                Find and buy the perfect car or truck from
```
Comment (RVMendoza) [inline html/components/footer.stories.js:128]: Period at the end

**Review by bobwise — APPROVED** [review]
Approved once those periods are added. Great job @guern1kn!

**Review by RVMendoza — APPROVED** [review]

**Pull request closed without merging**